src/main/kotlin/exercise4/task1/BankAccount.kt

```diff
@@ -32,21 +32,52 @@ package exercise4.task1
  *      The initial balance for this constructor should always be set to 0.
  */
 
+open class BankAccount(
+    private val accountNumber: String,
+    private val accHolderName: String,
+    private var balance: Double
+) {
+    constructor(accountNumber: String, accHolderName: String): this(accountNumber, accHolderName, 0.0)
+
+    open fun deposit(amount: Double): Boolean {
+        balance += amount
+        return true
+    }
+
+    open fun withdraw(amount: Double): Boolean {
+        if (amount <= balance) {
+            balance -= amount
+            return true
+        }
+        return false
+    }
+
+    fun getBalance(): Double {
+        return balance
+    }
```

Comment:
If we change visibility of `balance` property to `protected` we wouldn't have to create the getter

Reply:
The task says to create getBalance method